src/main/kotlin/no/nav/syfo/application/VedtakService.kt

```diff
@@ -52,7 +53,8 @@ class VedtakService(
             behandlerMeldingPdf = behandlerMeldingPdf,
         )
 
-        // TODO: Publiserer behandlermelding på kafka til isdialogmelding
+        // TODO: How should we handle result here?
+        behandlerMeldingProducer.send(personident, createdBehandlerMelding, behandlerMeldingPdf)
```

Comment:
Usikker på om vi bør håndtere resultatet av denne sendingen på noen måte? 🤔

Reply:
Hvis sendingen feiler så burde kanskje kallet feile også?

Reply:
Ja, så da må man sjekke om result.success er true, ikke sant?

Reply:
Bare tenkte at kallet kanskje burde gjenspeile at lagring fungerte, men ikke produsering? 🤔 Eller at kallet egentlig bare går gjennom og vi får en error i loggene og en cronjob som rydder opp. Vet ikke helt 😅

Reply:
Evt går produseringen asynkront, spørs litt hva vi ønsker å gå for 🤷🏼‍♂️

Reply:
Hmja, flere muligheter her... Kunne gjort det synkront, men da må vi ha en mulighet for å rydde opp ved evt feil (og sørge for at vi plukker opp disse - alarm?) ved å f.eks. ha en cronjob som kan kjøres ved behov. Men hvis vi uansett må lage en cronjob så er det kanskje like greit å kjøre all produseringen asynkront? :man_shrugging: